crux-mir/README.md

```diff
@@ -19,6 +19,22 @@ Next, navigate to the `crucible/dependencies/mir-json` directory and install
 `mir-json` according to the instructions in [the `mir-json`
 README][mir-json-readme].
 
+Currently, `crux-mir` supports [version
+1](https://github.com/GaloisInc/mir-json/blob/master/SCHEMA_CHANGELOG.md#1) of
+`mir-json`'s schema. Note that the schema versions produced by `mir-json` can
+change over time as dictated by internal requirements and upstream changes. To
+help smooth this over, we intend that once:
+
+* `crux-mir` introduces support for any given schema version, it will retain
+  that support across at least two releases.
```

Comment:
Upon a closer read, this would be naturally structured as:

```suggestion
help smooth this over:

* We intend that once `crux-mir` introduces support for any given schema version, it will retain
  that support across at least two releases.
```